Assets/Scripts/DataController.cs

```diff
@@ -94,6 +105,11 @@ void Start()
                 cityDropdown.GetComponent<CityDropdown>().InitCityNames(cities, SelectedCity);
             }
         }
+        if (constellationAbbrData != null)
+        {
+            allConstellationAbbrs = DataImport.ImportConstellationAbbreviationData(constellationAbbrData.text);
+        }
+
```

Comment:
@dstrawberrygirl We could import this constellation data in another module (the new Utils class, for example).  We'll need to frequently look up a conversion from this data so it might be better placed elsewhere.  Thoughts?